### DISCUSSION TITLE
Webpack compile error

### DISCUSSION THREAD

**Discussion by kbatchu** [visgl/loaders.gl#2829] [Q&A] [answered]
Hello,
I am getting the following 2 errors when compiling with Webpack v 5.75.0 with Node v20.10.0 and npm v9.4.2

ERROR in ./node_modules/@loaders.gl/bson/dist/lib/encoders/encode-bson.js 3:21-30
export 'serialize' (imported as 'serialize') was not found in 'bson' (possible exports: BSONLoader, BSONWriter)
 @ ./node_modules/@loaders.gl/bson/dist/bson-writer.js 1:0-63 13:11-25 16:11-25
 @ ./node_modules/@loaders.gl/bson/dist/index.js 2:0-46 2:0-46
 @ ./node_modules/@loaders.gl/parquet/dist/parquetjs/schema/types.js 1:0-58 289:47-68 289:153-163 294:34-54 294:139-149
 @ ./node_modules/@loaders.gl/parquet/dist/parquetjs/schema/schema.js 4:0-51 98:20-41
 @ ./node_modules/@loaders.gl/parquet/dist/index.js 25:0-61 25:0-61

ERROR in ./node_modules/@loaders.gl/bson/dist/lib/parsers/parse-bson.js 3:21-32
export 'deserialize' (imported as 'deserialize') was not found in 'bson' (possible exports: BSONLoader, BSONWriter)
 @ ./node_modules/@loaders.gl/bson/dist/bson-loader.js 1:0-60 23:9-22 30:9-22
 @ ./node_modules/@loaders.gl/bson/dist/index.js 1:0-46 1:0-46
 @ ./node_modules/@loaders.gl/parquet/dist/parquetjs/schema/types.js 1:0-58 289:47-68 289:153-163 294:34-54 294:139-149
 @ ./node_modules/@loaders.gl/parquet/dist/parquetjs/schema/schema.js 4:0-51 98:20-41
 @ ./node_modules/@loaders.gl/parquet/dist/index.js 25:0-61 25:0-61

How do I fix this?
Thanks
Kiran

**Comment by ibgreen** [comment, marked as answer]
#2831 may fix this. An educated guess as I don't have a webpack 5 test rig. It should be published in 4.1.0-alpha.3 within a week

**Reply by kbatchu** [reply]
Thanks.

**Comment by kbatchu** [comment]
Hello,
When will 4.1.0 be available?
Thanks.
